docs/source/core.rst

```diff
@@ -107,6 +107,49 @@ It is recommended to use `useContractCall`_ where applicable instead of this met
 
 - ``calls: ChainCall[]`` - list of calls, also see `ChainCall`_. Calls need to be in the same order across component renders.
 
+.. _useTypedContractCall-label:
+
+useTypedContractCall
```

Comment:
Can we merge `useTypedContractCall` and `useContractCall` into one function that accepts to different Call types as arguments?